### DISCUSSION TITLE
Workaround Tailwind 3.3 not correctly applying dark styles

### DISCUSSION THREAD

**Discussion by doutatsu** [tailwindlabs/tailwindcss#11665] [Help]
I am curious if anyone else is having this issue - https://github.com/tailwindlabs/tailwindcss/issues/11024 - and found a workaround. I've been stuck with Tailwind 3.2 for awhile now, as I am unable to proceed without fixing the dark theming.

**Comment by bernadinerandall** [comment]
If issue #11024 is related to the Tailwind CSS GitHub repository, you can visit the repository's issues section to see if others have reported the same problem or if there are any updates from the maintainers or contributors. [Tell Popeyes](https://www.tellpopeyes.biz/)

**Reply by doutatsu** [reply]
If you have checked the issue, you would have seen that the conversation has concluded and there has been no follow-ups anymore, even when asked. Which is why I decided to post here in hopes more people would comment on the issue
